aws-ami.pkr.hcl

```diff
@@ -40,25 +46,64 @@ variable "build_subnet_id" {
   type        = string
 }
 
+variable "source_ami_name_filter_arm64" {
+  description = "Name filter for the source arm64 AMI image"
+  type        = string
+  default     = "ubuntu/images/hvm-ssd/ubuntu-*-22.04-arm64-server-*"
+}
+
+variable "source_ami_name_filter_x86-64" {
+  description = "Name filter for the source x86-64 AMI image"
+  type        = string
+  default     = "ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-*"
+}
+
 # Defines a local variable to generate a timestamp for AMI naming.
 locals {
   timestamp = regex_replace(timestamp(), "[- TZ:]", "")
 }
 
 # Defines the source configuration for the Amazon EBS builder.
-source "amazon-ebs" "ubuntu" {
-  ami_name      = "ami-ontotext-graphdb-${var.gdb_version}-${local.timestamp}"
-  instance_type = "${var.build_instance_type}"
+source "amazon-ebs" "ubuntu-x86-64" {
+  ami_name      = "ami-ontotext-graphdb-${var.gdb_version}-x86-64-${local.timestamp}"
```

Comment:
Btw, not sure but I think we should use `_` for `x86_64`

Reply:
https://en.wikipedia.org/wiki/X86-64